<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
        <item>
         <widget class="QLabel" name="hedding">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 700 18pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>Stenography</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3">
          <item>
           <widget class="QTreeWidget" name="Options">
            <property name="styleSheet">
             <string notr="true">QTreeWidget::item{
	font: 700 15pt &quot;Ubuntu Mono&quot;;
	border-radius:2px;
	padding-top:10px;
	
}
</string>
            </property>
            <column>
             <property name="text">
              <string>Opertions</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>Image Encode</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Image to Image Encode</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Image exe Encode</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="page_2">
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QTabWidget" name="tabWidget_3">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <widget class="QWidget" name="widget">
                 <attribute name="title">
                  <string>Image exe Encode</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_9">
                  <item row="0" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_12">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_12">
                      <item>
                       <widget class="QPushButton" name="uploadexe_btn">
                        <property name="text">
                         <string>Upload exe</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="uploadexe_line"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_11">
                      <item>
                       <widget class="QPushButton" name="uploadimg_btn">
                        <property name="text">
                         <string>Upload Image</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="uploadimage_line"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_15">
                      <item>
                       <widget class="QPushButton" name="processimgexe">
                        <property name="text">
                         <string>Process</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_7">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_2">
                 <attribute name="title">
                  <string>Image exe Decode</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="0" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_13">
                      <item>
                       <widget class="QPushButton" name="decodeexe_btn">
                        <property name="text">
                         <string>Decode</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="decodeexex_line"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_14">
                      <item>
                       <widget class="QPushButton" name="processdecodeexe">
                        <property name="text">
                         <string>Process</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_8">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_0">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QTabWidget" name="tabWidget">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="img_encode">
                 <attribute name="title">
                  <string>Encode Text</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,4,1">
                      <item>
                       <widget class="QLabel" name="label">
                        <property name="text">
                         <string>Encode Image With Text</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                         <widget class="QLineEdit" name="path_encode_img"/>
                        </item>
                        <item>
                         <widget class="QPushButton" name="open_encode_img">
                          <property name="text">
                           <string>Open</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_4">
                        <item alignment="Qt::AlignHCenter">
                         <widget class="QLabel" name="img_enode_holder">
                          <property name="minimumSize">
                           <size>
                            <width>350</width>
                            <height>0</height>
                           </size>
                          </property>
                          <property name="maximumSize">
                           <size>
                            <width>400</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="frameShadow">
                           <enum>QFrame::Raised</enum>
                          </property>
                          <property name="text">
                           <string/>
                          </property>
                          <property name="pixmap">
                           <pixmap>../../../../../../../Pictures/WhatsApp Image 2023-11-03 at 12.21.56 AM.jpeg</pixmap>
                          </property>
                          <property name="scaledContents">
                           <bool>true</bool>
                          </property>
                          <property name="wordWrap">
                           <bool>false</bool>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_3">
                        <property name="bottomMargin">
                         <number>15</number>
                        </property>
                       </layout>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_4">
                      <item>
                       <spacer name="horizontalSpacer">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QPushButton" name="pushButton">
                        <property name="text">
                         <string>Export</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="img_decode">
                 <attribute name="title">
                  <string>Decode Text</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_10">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0,4,1">
                      <item>
                       <widget class="QLabel" name="label_2">
                        <property name="text">
                         <string>Decode Image</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_5">
                        <item>
                         <widget class="QPushButton" name="img_decode_open">
                          <property name="text">
                           <string>Open</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <spacer name="horizontalSpacer_4">
                          <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                           <size>
                            <width>40</width>
                            <height>20</height>
                           </size>
                          </property>
                         </spacer>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_8">
                        <item>
                         <widget class="QLabel" name="img_decode_holder">
                          <property name="minimumSize">
                           <size>
                            <width>350</width>
                            <height>0</height>
                           </size>
                          </property>
                          <property name="text">
                           <string/>
                          </property>
                          <property name="scaledContents">
                           <bool>true</bool>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_9">
                        <item>
                         <widget class="QTextEdit" name="decode_img_text"/>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_1">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QTabWidget" name="tabWidget_2">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <widget class="QWidget" name="tab_3">
                 <attribute name="title">
                  <string>Encode</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0,0,2,0">
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Hide Image inside Image</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_9">
                      <item>
                       <widget class="QPushButton" name="primary_img">
                        <property name="text">
                         <string>Primary Image</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_5">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeType">
                         <enum>QSizePolicy::Minimum</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QPushButton" name="seconday_img">
                        <property name="text">
                         <string>Seconday Image</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <property name="spacing">
                       <number>15</number>
                      </property>
                      <property name="topMargin">
                       <number>25</number>
                      </property>
                      <property name="bottomMargin">
                       <number>25</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="img_img_primary">
                        <property name="text">
                         <string>Primary Image</string>
                        </property>
                        <property name="scaledContents">
                         <bool>true</bool>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignCenter</set>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="img_img_secondary">
                        <property name="text">
                         <string>Seconday Image</string>
                        </property>
                        <property name="scaledContents">
                         <bool>true</bool>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignCenter</set>
                        </property>
                        <property name="wordWrap">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_7">
                      <item>
                       <widget class="QPushButton" name="img_img_export">
                        <property name="text">
                         <string>Export</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_2">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_4">
                 <attribute name="title">
                  <string>Decode</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="0" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_15" stretch="0,0,2,0">
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Decode Hiddin Image</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_10">
                      <item>
                       <widget class="QPushButton" name="open_encode_img_2">
                        <property name="text">
                         <string>Open</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_6">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_14">
                      <property name="leftMargin">
                       <number>25</number>
                      </property>
                      <property name="topMargin">
                       <number>25</number>
                      </property>
                      <property name="rightMargin">
                       <number>25</number>
                      </property>
                      <property name="bottomMargin">
                       <number>25</number>
                      </property>
                      <item alignment="Qt::AlignHCenter">
                       <widget class="QLabel" name="img_img_decode">
                        <property name="minimumSize">
                         <size>
                          <width>350</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Img_img_decode</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_8">
                      <item>
                       <widget class="QPushButton" name="pushButton_2">
                        <property name="text">
                         <string>Export</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_3">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Options</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>path_encode_img</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>img_decode_open</tabstop>
  <tabstop>decode_img_text</tabstop>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>primary_img</tabstop>
  <tabstop>img_img_export</tabstop>
  <tabstop>open_encode_img_2</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
